src/Nethermind/Nethermind.Runner/Runners/EthereumRunner.cs

```diff
@@ -949,7 +947,7 @@ private async Task InitPeer()
             _staticNodesManager = new StaticNodesManager(_initConfig.StaticNodesPath, _logManager);
             await _staticNodesManager.InitAsync();
 
-            var peersDb = new SimpleFilePublicKeyDb("PeersDB", Path.Combine(_executingDirectory, _initConfig.BaseDbPath, PeersDbPath), _logManager);
+            var peersDb = new SimpleFilePublicKeyDb("PeersDB", Path.Combine(_initConfig.BaseDbPath.GetApplicationResourcePath(), PeersDbPath), _logManager);
```

Comment:
My intended usage was PeersDbPath.GetApplicationResourcePath(_initConfig.BaseDbPath), but it may lead to same results, though abstracting everything in one place